packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx

```diff
@@ -89,7 +70,7 @@ export const PlaylistLibrary = (props: PlaylistLibraryProps) => {
         <CreatePlaylistLibraryItemButton scrollbarRef={scrollbarRef} />
       </GroupHeader>
       {!library || isEmpty(library?.contents) ? (
-        <EmptyLibraryNavLink onClick={handleCreatePlaylist} />
+        <EmptyLibraryNavLink />
```

Comment:
cant remember which component this is, but do we want to do something else on click, or remove click interaction all together?

Reply:
the click is handled in the component itself now!